test/user.spec.js

```diff
@@ -52,3 +58,42 @@ describe('User Creation', () => {
         expect(res.body.message).toEqual('The name cannot be empty');
       }));
 });
+
+describe('User Sign-In', () => {
+  it('Responds with success when the user is logged in with the correct credentials', () =>
+    request(app)
+      .post('/users')
+      .send(user)
+      .then(() =>
+        request(app)
+          .post('/users/sessions')
+          .send(signIn)
+          .then(res => {
+            expect(res.statusCode).toEqual(200);
+            expect(jwt.decode(res.body.token, secret).username).toEqual(signIn.email);
```

Comment:
we prefer using strict equal

Reply:
I used strictEqual and toBe, it seems that toBe works better for testing primitive types :) https://stackoverflow.com/questions/45195025/what-is-the-difference-between-tobe-and-toequal-in-jest